src/transactions/feed/TransactionFeed.test.tsx

```diff
@@ -16,9 +16,9 @@ import {
 } from 'src/transactions/types'
 import networkConfig from 'src/web3/networkConfig'
 import { createMockStore, RecursivePartial } from 'test/utils'
-import { mockCusdAddress, mockCusdTokenId } from 'test/values'
-import { mockApprovalTransaction } from 'test/values'
+import { mockApprovalTransaction, mockCusdAddress, mockCusdTokenId } from 'test/values'
 
+jest.mock('src/jumpstart/utils')
```

Comment:
perhaps we can mock the return value of statsig rather than mocking this module? there are no unit tests for this module so kind of nice to not exclude this file intentionally